### ISSUE TITLE
primer length values are not passed through to figaro

### ISSUE THREAD

**Issue by cschu** [cschu/gaga2#2]
```
Command error:
  usage: figaro [-h] [-o OUTPUTDIRECTORY] -a AMPLICONLENGTH -f FORWARDPRIMERLENGTH -r REVERSEPRIMERLENGTH [-i INPUTDIRECTORY] [-n OUTPUTFILENAME] [-m MINIMUMOVERLAP] [-s SUBSAMPLE] [-p PERCENTILE] [-F FILENAMINGSTANDARD] [-l LOGFILE]
  figaro: error: argument -f/--forwardPrimerLength: invalid int value: 'null'

Work dir:
  [...]

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`

./test_input.sh: line 6: --left_primer: command not found
```

**Comment by cschu** [comment]
1. different external / internal parameter names for primer lengths: `--left_primer/--right_primer` vs `params.forward_primer/params.reverse_primer`
2. `./test_input.sh: line 6: --left_primer: command not found` was caused by typo in submission script (blank after line-wrap `\`)

**Comment by cschu** [comment]
issue addressed in 651d41f
